packages/backend/README.md

```diff
@@ -37,13 +29,14 @@ docker exec -it state_explorer_postgres psql -U postgres -c 'CREATE DATABASE loc
 docker exec -it state_explorer_postgres psql -U postgres -c 'CREATE DATABASE test'
 ```
 
-If you restart your system running `docker start state_explorer_postgres` will bring the database back online.
+If you restart your system, running `docker start state_explorer_postgres` will bring the database back online.
 
-Once you have everything create a `.env` file with the following contents:
+Once you have everything, create a `.env` file with the following contents:
 
 ```
 LOCAL_DB_URL=postgresql://postgres:password@localhost:5432/local
 TEST_DB_URL=postgresql://postgres:password@localhost:5432/test
+LOCAL_JSON_RPC_URL=https://eth-mainnet.alchemyapi.io/v2/zR5UQGm__S5p1pVRUG3vCsK5elOgJL6v
```

Comment:
Don't push secrets to the repository :) Instead you can say `<PASTE_YOUR_ALCHEMY_URL_HERE>`